README.md

```diff
@@ -10,10 +10,20 @@
 
 ## Videos 🎥
 
+- [Pavel Pevzner explains short read assembling] (https://www.youtube.com/watch?feature=player_embedded&list=PLQ-85lQlPqFNGdaeGpV8dPEeSm3AChb6L&v=vjB6nhOu3BY)
```

Comment:
the part `feature=player_embedded&list=PLQ-85lQlPqFNGdaeGpV8dPEeSm3AChb6L&` can be removed